### PR TITLE
Add dependencies to setup.py

### LINKED ISSUE #61: Missing dependencies while installing from PyPI
```
$ pip install repolib
Collecting repolib
  Using cached repolib-1.5.1-py3-none-any.whl (65 kB)
Installing collected packages: repolib
Successfully installed repolib-1.5.1
$ pip freeze
repolib==1.5.1
```

### PR DESCRIPTION
**Pull request by drnextgis**
Fixes #61

### CODE REVIEW

**Comment by MestreLion** [comment]
`dbus-python` is missing from this list.

Also, I believe `pytest-pylint` and `python-debian` should be under _build_ requirements, not install. Are they actually needed when _using_ the library or running `apt-manage`?

**Comment by MestreLion** [comment]
Also, installing `python-dbus` from PyPi requires some system packages that must be installed via package manager. This is not clearly documented by https://dbus.freedesktop.org/doc/dbus-python/ : they only mention _"it uses `libdbus` ..."_, but there are no install instructions or requirements declared. I was able to pull the requirements from their [CI scripts](https://gitlab.freedesktop.org/dbus/dbus-python/-/blob/master/tools/ci-install.sh):
```
sudo apt install libdbus-1-dev
```
(possibly other apt-packages are needed, my system already has a ton of dev/build-related packages installed, so a clean system would be required to list them all)

I believe this should be mentioned in the `repolib`'s documentation

**Comment by MestreLion** [comment]
By the way... `python-debian` is indeed needed at run time, so keep it in the list. (`launchpadlib` will pull it anyway, as it also depends on it.)

And I just tested in a virtualenv, and `pytest-pylint` is _not_ needed at run time, so it should go to `setup_requires` instead of `install_requires`.

Meanwhile, this PR could also delete `requirements.txt`: that's not meant to be commited to a source repository. (and it's also missing `launchpadlib` and `gnupg`, which are run-time dependencies)

**Comment by drnextgis** [comment]
@MestreLion thanks for looking at the PR. I've update it according to your comments.

**Comment by MestreLion** [comment]
@drnextgis you're welcome! I've just found this library, it's exactly what I was looking for (was about to create one for PPAs), and your PR is a much needed addition, so I thank you for this!

File: setup.py [hunk 1]
```diff
@@ -183,6 +183,14 @@ def run(self):
     download_url='https://github.com/pop-os/repolib/releases',
     long_description=long_description,
     tests_require=['pytest'],
+    setup_requires=['pytest-pylint'],
+    install_requires=[
+        'dbus-python',
+        'distro',
+        'gnupg',
+        'launchpadlib',
```
Comment (isantop) [inline setup.py:190]: `launchpadlib` isn't something that's a hard-requirement, as it's only used for enabling PPA support. The rest of the library still functions without it, though installing it does enable the PPA support. Is adding it in `install_requires` still appropriate?

Comment (drnextgis) [reply setup.py:190]: PPASource is being imported in [\_\_init\_\_.py](https://github.com/pop-os/repolib/blob/f99074ded98e832c62931ed3ab64f0bf62f72e9c/repolib/__init__.py#L34) so whenever we import `repolib` it fails if `launchpadlib` is not installed:
```
>>> import repolib
Traceback (most recent call last):
  File "/home/denis/git/repolib/repolib/shortcuts/ppa.py", line 32, in <module>
    from launchpadlib.launchpad import Launchpad
ModuleNotFoundError: No module named 'launchpadlib'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/denis/git/repolib/repolib/__init__.py", line 34, in <module>
    from .shortcuts import PPASource, PopdevSource, shortcut_prefixes
  File "/home/denis/git/repolib/repolib/shortcuts/__init__.py", line 24, in <module>
    from .ppa import PPASource
  File "/home/denis/git/repolib/repolib/shortcuts/ppa.py", line 36, in <module>
    'Missing optional dependency "launchpadlib". Try `sudo apt install '
repolib.source.SourceError: Missing optional dependency "launchpadlib". Try `sudo apt install python3-launchpadlib` to install it.
```

Comment (isantop) [reply setup.py:190]: I see. I think I'd prefer to fix that issue, but for now that should be okay

**Review by isantop — APPROVED** [review]

**Review by n3m0-22 — APPROVED** [review]
Looks good. Working as expected.

**Pull request merged**